octoprint_mrbeam/migration/Mig006.py

```diff
@@ -39,106 +36,24 @@ def shouldrun(cls, beamos_version):
 
         overrides the current behaviour as this migration should run if the log file contains the "octoprint.plugins.mrbeam.analytics.usage - ERROR - No job time found in {}, returning 0" error
         """
-        command_output, code = exec_cmd_output(
-            Mig006FixUsageData.COMMAND_TO_CHECK_IF_VERSION_WAS_PRESENT,
-            log=True,
-            shell=True,
-        )
+        with open(cls.USAGE_DATA_FILE_PATH, "r") as yaml_file:
+            yaml_data = yaml.load(yaml_file, Loader=OrderedLoader)
+            if "prefilter" in yaml_data or "carbon_filter" in yaml_data:
+                return True
```

Comment:
why not back up all files regardless if the key exist or not?

Reply:
Then it would trigger for new devices as well, this will make sure it will only trigger when the migration for the new structure is needed, newly produced devices will not need this